src/index.html

```diff
@@ -1,13 +1,272 @@
+          </div>
+
+          <h4 class="title title--level4 item-box__name">SoundLink Color Bluetooth speaker&nbsp;II</h4>
+
+          <div class="item-box__desc">
+            Portable bluetooth
+          </div>
+
+          <div class="item-box__price">
+            $ 129.00
+          </div>
+        </article>
+      </section>
+
+      <section class="product-categories main-section__section" id="categories">
+        <h2 class="title title--level2 title--center product-categories__title">Browse Bose<br> products by category</h2>
+
+        <article class="single-category product-categories__item">
+          <div class="single-category__gallery">
+            <img
+              src="images/photos/cat-image1.jpg"
+              alt="Woman listening to music in headphones"
+              class="single-category__image single-category__image--big"
+            >
+            <img
+              src="images/photos/cat-image2.jpg"
+              alt="Woman taking pictures in headphones"
+              class="single-category__image single-category__image--small"
+            >
+          </div>
+
+          <h4 class="title title--level4 single-category__title">Headphones & earbuds</h4>
+        </article>
+
+        <article class="single-category product-categories__item">
+          <div class="single-category__gallery">
+            <img
+              src="images/photos/cat-image3.jpg"
+              alt="Person connecting speaker to clip hook"
+              class="single-category__image single-category__image--big"
+            >
+            <img
+              src="images/photos/cat-image4.jpg"
+              alt="Blue speaker"
+              class="single-category__image single-category__image--small"
+            >
+          </div>
+
+          <h4 class="title title--level4 single-category__title">Speakers</h4>
+        </article>
+
+        <article class="single-category product-categories__item">
+          <div class="single-category__gallery">
+            <img
+              src="images/photos/cat-image5.jpg"
+              alt="Man in sunglasses"
+              class="single-category__image single-category__image--small"
+            >
+            <img
+              src="images/photos/cat-image6.jpg"
+              alt="Woman in sunglasses"
+              class="single-category__image single-category__image--big"
+            >
+          </div>
+
+          <h4 class="title title--level4 single-category__title">Audio sunglasses</h4>
+        </article>
+
+        <a href="#" class="btn btn--large product-categories__button">View all</a>
+      </section>
+
+      <section class="why-buy main-section__section" id="how-to-buy">
+        <h2 class="title title--level2 why-buy__title">Why buy direct from Bose</h2>
+
+        <ul class="why-buy__list list">
+          <li><span class="title title--level4">Free 2-day shipping and returns</span></li>
```

Comment:
Not sure if you semantically need those spans inside li's here, I would personally just put text inside li without those spans